Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.6
+
+WORKDIR /app
+COPY . /app
```

Comment:
top tip: move the `COPY` to the bottom of the file to increase layer re-use
as then the only layer that has changed is the one with the code in

Reply:
correct, but both of those lines can be under `EXPOSE` and `CMD`
generally what i tend to do it copy just the `requirements.txt` and then `RUN` then copy the rest of the code after.
That way the container is really quick to build if the requirements don't change 
so something like 
```
FROM python:3.6
WORKDIR /app
EXPOSE 8086
CMD  ["python3","main.py"] 
COPY requirements.txt /app/requirements.txt
RUN pip3 install --require-hashes -r requirements.txt
COPY . /app
```

Reply:
Okay I'll put that in 👍